examples/advanced.ts

```diff
@@ -123,6 +129,55 @@ async function concurrentGetsExample() {
   });
 }
 
+async function middlewaresExample() {
+  const middlewaresExampleloggerFactory: MomentoLoggerFactory =
+    new DefaultMomentoLoggerFactory(DefaultMomentoLoggerLevel.DEBUG);
+  const middlewaresExamplelogger = middlewaresExampleloggerFactory.getLogger(
+    'AdvancedMiddlewaresExample'
+  );
+  middlewaresExamplelogger.info(
+    'Constructing a new Momento client with logging and metrics middlewares enabled'
+  );
+
+  const metricsCsvPath = './advanced-middlewares-example-metrics.csv';
```

Comment:
Is this checked in? I can't seem to find it.

Reply:
no, i didn't intend to check it in, it changes every run.  You can see an example of what it looks like in the other pr.